### ISSUE TITLE
Popup message detect URL links

### ISSUE THREAD

**Issue by superh2m** [thecodingmachine/workadventure#984]
Hello again ^^

It will be great to detect URL links in popup message, and wrap them to create `<a>` links

![image](https://user-images.githubusercontent.com/6178360/116789701-61ff0100-aab0-11eb-9922-031d7b36f773.png)

![image](https://user-images.githubusercontent.com/6178360/116786245-e1cfa000-aa9d-11eb-907f-9d16d77bea39.png)

**Comment by jonnytest1** [comment]
openTab(url: string): void;
 goToPage(url: string): void;
 
 though you would have to add an "open" option to your popup

**Comment by superh2m** [comment]
Yes indeed, but I don't want to trigger the action instead of the user
I wanted to have clickable links in the popup messages

I know that buttons can do the job, but it can be nice also to be able to click directly on links in the popup messages, like chat messages

**Comment by jonnytest1** [comment]
i jsut realized writing https://www.google.de in chat actually does provide a link

**Comment by superh2m** [comment]
so let's use `HtmlUtils.urlify(message)` everywhere :partying_face:

**Comment by jonnytest1** [comment]
@moufmouf how about it?
